packages/patrol/test/patrol_tester_test.dart

```diff
@@ -936,5 +937,104 @@ void main() {
         expect($('bottom text').visible, true);
       });
     });
+
+    group('pumpAndMaybeSettle()', () {
+      late int count;
+      late int state;
+
+      setUp(() {
+        count = 0;
+        state = 0;
+      });
+
+      final appWithInfiniteAnimation = MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: StatefulBuilder(
+              builder: (context, setState) {
+                return Column(
+                  children: [
+                    CircularProgressIndicator(),
+                    Text('count: $count'),
+                    ElevatedButton(
+                      onPressed: () => setState(() => count++),
+                      style: ElevatedButton.styleFrom(
+                        textStyle: const TextStyle(fontSize: 20),
+                      ),
+                      child: const Text('Enabled button'),
+                    ),
+                  ],
+                );
+              },
+            ),
+          ),
+        ),
+      );
+
+      final appWithAnimationOnTap = MaterialApp(
```

Comment:
```suggestion
      final appWithInfiniteAnimationOnTap = MaterialApp(
```

Reply:
That's not true, the animation is finite